app/src/main/java/org/merakilearn/ui/onboarding/OnBoardPagesAdapter.kt

```diff
@@ -45,14 +45,14 @@ class OnBoardPagesAdapter(
     }
 
     override fun getItemCount(): Int {
-        return onBoardingData.onBoardingPagesList.size
+        return onBoardingData.onBoardingPagesList.size - 1
```

Comment:
Why are we subtracting 1?

Reply:
Actually we want to have only 3 pages at the time of onboarding that is why subtracted 1 so that we can have 3 pages in place of 4 pages by default and we tried to do this other way but that is causing certain errors in remote cofig inside core module